runtime/src/bank.rs

```diff
@@ -7839,8 +7855,7 @@ impl Bank {
 
     pub(crate) fn do_fill_bank_with_ticks_for_tests(&self, scheduler: &InstalledSchedulerRwLock) {
         if self.tick_height.load(Relaxed) < self.max_tick_height {
-            let last_blockhash = self.last_blockhash();
-            while self.last_blockhash() == last_blockhash {
+            while self.tick_height.load(Relaxed) < self.max_tick_height {
```

Comment:
Change here because `register_tick` no longer directly changes the result of `last_blockhash()`. Instead of looping until a change, we loop until we reach the max tick height.